### DISCUSSION TITLE
Question 2

### DISCUSSION THREAD

**Discussion by Sreekar2610** [Sreekar2610/python#6] [Q&A] [answered]
why python is most popular programming language as compared to other programming languages like C , JAVA etc... ?

**Comment by Maithri1234** [comment, marked as answer]
Python empowers developers to employ a variety of programming styles while they're creating programs. Because it's so flexible, you might use it, not just for object-oriented programming, but also for functional and reflective programming. It's popular among programmers for back-end development and app development.
